tests/Unit/Service/SubmissionServiceTest.php

```diff
@@ -632,6 +637,42 @@ public function dataValidateSubmission() {
 				// Expected Result
 				false
 			],
+			'invalid-short-phone' => [
+				// Questions
+				[
+					['id' => 1, 'type' => 'short', 'isRequired' => false, 'extraSettings' => ['type' => 'tel']]
```

Comment:
```suggestion
					['id' => 1, 'type' => 'short', 'isRequired' => false, 'extraSettings' => ['validationType' => 'tel']]
```
Shouldn't the parameters be `validationType` instead of `type`

Reply:
Rebase issue :see_no_evil: